### DISCUSSION TITLE
How to specify data values when using Create Row or Update Row option

### DISCUSSION THREAD

**Discussion by vikrantsharma123** [Budibase/budibase#3126] [Help]
Hi all,
When creating an automation, I am trying to use either the Create Row or Update Row option, but I am not able to see how to specify the column values for the row. 

Please suggest. 

Regards,
Vikrant Sharma

**Comment by mjashanks** [comment]
Hi Vikrant.

My Create Row action displays the available columns, once I select the table:

![image](https://user-images.githubusercontent.com/3524181/138547086-bf934699-9d01-4f51-82e0-580c29ed6a18.png)

Are all of your columns missing? what database are you using? Budibase built-in DB or SQL?

**Reply by vikrantsharma123** [reply]
Hi Michael,
I am using Budibase built-in DB. My "Business Unit Audit Log" table has only a "Changes" column (long form text) which I would like to fill, when a new Business Unit is created. But I do not see that column when I select the table. Please check this screenshot.

![image](https://user-images.githubusercontent.com/14836167/138549569-4ef282a2-fbee-49ec-b7da-3eeb2472de4e.png)

**Reply by vikrantsharma123** [reply]
Hi Michael,
If I make a simple Text column, it does show up in the Create Row action of automate. The "long form text" column does not show up though. Maybe that is the issue.

Regards,
Vikrant Sharma

**Reply by mjashanks** [reply]
Ah yes, I just realised that there is an open issue that we have fixed but not released.

https://github.com/Budibase/budibase/issues/3006

This will be released next week.

**Reply by vikrantsharma123** [reply]
Thanks Michael.
